strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/layout/AbstractLayoutProvider.java

```diff
@@ -135,12 +135,16 @@ public final ArtifactOutputStream getOutputStream(RepositoryPath path) throws IO
     
     @Override
     public URI resolveResource(Repository repository,
-                               String path) throws IOException
+                               String path) 
+                throws IOException,
+                       IllegalArgumentException
```

Comment:
I don't think any change to this file is necessary. Hence, additional `throws` declaration of `IllegalArgumentException` here is also unnecessary. Would you mind reverting this file to the previous state so there won't be any noise in the Pull Request related with this file ?